Library/Formula/let-me-in.rb

```diff
@@ -0,0 +1,36 @@
+class LetMeIn < Formula
+  desc "Add my IP to AWS security group(s)"
+  homepage "https://github.com/rlister/let-me-in"
+  url "https://github.com/rlister/let-me-in/archive/v0.0.3.tar.gz"
+  sha256 "538ada5748d513a3a926d7795ddcbe13bf96c99e0921cdc4148ff38f0d196121"
+
+  depends_on "go" => :build
+
+  ## this works
+  go_resource "github.com/aws/aws-sdk-go/aws" do
+    url "github.com/aws/aws-sdk-go/aws.git", revision: 'f096b7d61df3d7d6d97f0e701f92616d1ea5420d'
+  end
+
+  ## this works
+  go_resource "github.com/aws/aws-sdk-go/aws/awserr" do
+    url "github.com/aws/aws-sdk-go/aws.git", revision: 'f096b7d61df3d7d6d97f0e701f92616d1ea5420d'
+  end
+
+  ## this DOES NOT work
+  go_resource "github.com/aws/aws-sdk-go/service/ec2" do
+    url "github.com/aws/aws-sdk-go/aws.git", revision: 'f096b7d61df3d7d6d97f0e701f92616d1ea5420d'
+  end
+
+  def install
+    ENV["GOPATH"] = buildpath
+    # system "go", "get", "github.com/aws/aws-sdk-go/aws"
```

Comment:
You need to stage the resources in the buildpath as well. They won't automatically know where they're supposed to be and hop over. Something like this is fairly usual:

```ruby
    ENV["GOPATH"] = buildpath
    Language::Go.stage_deps resources, buildpath/"src"
```